pubsub/pubsub.go

```diff
@@ -208,7 +217,10 @@ type Subscription struct {
 // concurrently from multiple goroutines. The Ack() method of the returned
 // Message has to be called once the message has been processed, to prevent it
 // from being received again.
-func (s *Subscription) Receive(ctx context.Context) (*Message, error) {
+func (s *Subscription) Receive(ctx context.Context) (_ *Message, err error) {
+	ctx = trace.StartSpan(ctx, "gocloud.dev/pubsub.Receive")
```

Comment:
Shouldn't it be pubsub.Subscription.Receive?

Reply:
I don't feel strongly, but I also found it a bit weird that some had `Topic`/`Subscription` and others didn't. Might be better to be consistent even if it's not necessary to be unique.

Reply:
OK, wrote method path names out in full.